src/java/org/apache/cassandra/db/compaction/UnifiedCompactionContainer.java

```diff
@@ -67,8 +69,14 @@ public class UnifiedCompactionContainer implements CompactionStrategyContainer
 
         factory.getCompactionLogger().strategyCreated(this.strategy);
 
-        if (this.strategy.getOptions().isLogAll())
+        if (this.strategy.getOptions().isLogAll()){
```

Comment:
Please make sure the Cassandra code style is followed. See https://cassandra.apache.org/_/development/index.html, Code Style section. In particular, an opening bracket should be on the next line.